api/python/quilt3/backends/__init__.py

```diff
@@ -0,0 +1,19 @@
+from quilt3.util import PhysicalKey, get_from_config, fix_url
+
+from .base import PackageRegistry
+from .local import LocalPackageRegistryV1
+from .s3 import S3PackageRegistryV1
+
+
+def get_package_registry(path=None) -> PackageRegistry:
```

Comment:
Can we keep get_package_registry private (used only internal to the library)? If we make it public, it looks like we're duplicating a lot of API surface (and hiding telemetry in the process).